### PR TITLE
Fix import for bd subnets (DCNE-213)

### PR DESCRIPTION
**Pull request by juchowan**

### CODE REVIEW

**Review by akinross — APPROVED** [review]
LGTM

File: website/docs/r/schema_template_bd_subnet.html.markdown [hunk 1]
```diff
@@ -53,5 +53,5 @@ No attributes are exported.
 An existing MSO Schema Template Bridge Domain Subnet can be [imported][docs-import] into this resource via its Id/path, via the following command: [docs-import]: <https://www.terraform.io/docs/import/index.html>
 
 ```bash
-terraform import mso_schema_template_bd_subnet.bdsub1 {schema_id}/template/{template_name}/bd/{bd_name}/subnet/{ip}
+terraform import mso_schema_template_bd_subnet.bdsub1 {schema_id}/template/{template_name}/bd/{bd_name}/ip/{ip}
```
Comment (anvitha-jain) [inline website/docs/r/schema_template_bd_subnet.html.markdown:56]: shouldn't this be changed at website/docs/r/schema_site_bd_subnet.html.markdown as well?
Also test the changes made to import in terraform-provider-mso/examples/schema_site_bd_subnet

**Review by anvitha-jain — CHANGES_REQUESTED** [review]

File: website/docs/r/schema_template_bd_subnet.html.markdown (hunk 1 above)
Comment (anvitha-jain) [reply website/docs/r/schema_template_bd_subnet.html.markdown:56]: I understand the scenario, it looks good. I was just trying to verify the changes.

**Review by anvitha-jain — APPROVED** [review]
LGTM

**Review by samiib — APPROVED** [review]
LGTM

**Review by sajagana — APPROVED** [review]
LGTM!

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**